inference-engine/thirdparty/clDNN/src/kernel_selector_helper.cpp

```diff
@@ -734,6 +734,7 @@ void set_params(const program_node& node, kernel_selector::params& params) {
     const auto& program = node.get_program();
     const auto& device_info = program.get_engine().get_device_info();
 
+    params.uniqueID = std::to_string(program.get_id()) + "_"  + node.get_unique_id();
```

Comment:
I suggest to try to move this line to `set_optional_param()` function. Also change `program_impl` parameter with `program_node` in `set_optional_param()` function

Reply:
As I mentioned in [previous comment](https://github.com/openvinotoolkit/openvino/pull/6261#discussion_r662822925),
I try to pass unqiue_id through Params and options_params without interface change because concerns about extensive code changes. however, it is available to use option 2 or 3 if it is okay to change interface.

Reply:
@Andrei Molotkov, I modified the code to keep in Params and pass the Params to the GetEntryPoint by the comments.
Please review this PR again. thanks.